skimage/future/graph/graph_cut.py

```diff
@@ -195,10 +195,19 @@ def get_min_ncut(ev, d, w, num_cuts):
         The value of the minimum ncut.
     """
     mcut = np.inf
+    mn = ev.min()
+    mx = ev.max()
+
+    # If all values in `ev` are equal, it implies that the graph can't be
+    # further sub-divided. In this case the bi-partition is the the graph
+    # itself and an empty set.
+    min_mask = np.zeros_like(ev, dtype=np.bool)
+    if np.allclose(mn, mx):
+        return min_mask, mcut
```

Comment:
Thanks for this fix! Last question: is `mcut = np.inf` the correct thing to return here? (My gut says yes but I'm double-checking since you're more familiar with the algorithm. =)